xla/pjrt/pjrt_client.h

```diff
@@ -1011,6 +1020,37 @@ class PjRtClient {
         "DmaUnmap not supported on platform %s", platform_name()));
   }
 
+  // CrossHostSendBuffers, CrossHostReceiveBuffers, and CancelCrossHostTransfer
+  // are part of the new cross-host transfers API.
+
+  // Send buffers to remote devices specified by dst_global_device_ids.
+  virtual absl::StatusOr<std::vector<Future<>>> CrossHostSendBuffers(
+      absl::Span<PjRtBuffer* const> buffers,
+      absl::Span<const PjRtGlobalDeviceId> dst_global_device_ids,
+      std::vector<CrossHostTransferKey> transfer_keys) {
```

Comment:
Just to make sure I'm understanding, `buffers`, `dst_global_device_ids`, and `transfer_keys` should all be the same length? And `CrossHostSendBuffers` will send `buffers[i]` to `dst_global_device_ids[i]` with unique key `transfer_keys[i]`. And the receiving device has to call `CrossHostReceiveBuffers` before the call to `CrossHostSendBuffers`?